tests/integration/aws/test_credentials.py

```diff
@@ -13,7 +13,7 @@
 
 
 @pytest.fixture
-def run_migration(ws, sql_backend, env_or_skip):
+def run_migration(ws, sql_backend, env_or_skip, runtime_ctx):
```

Comment:
```suggestion
def run_migration(ws, sql_backend, env_or_skip, aws_cli_ctx):
```
Create aws cli context that skips on the absence of AWS profile